api-report/container-runtime.api.md

```diff
@@ -345,10 +352,13 @@ export interface IGCRuntimeOptions {
 
 // @public
 export interface IGCStats {
+    blobCount: number;
```

Comment:
I assume this is about attachement blobs, and if so, can you please name all related variables to have "attachement" work in them to differentiate vs. summary blobs?

Reply:
done.